docs/programming_guide/execution_api_type/client_api.rst

```diff
@@ -133,7 +133,7 @@ Below is a table overview of key Client APIs.
      - API Doc Link
    * - patch
      - Patches the PyTorch Lightning Trainer for usage with FLARE.
-     - :func:`train<nvflare.app_opt.lightning.api.patch>`
```

Comment:
Let me update the MONAI config and 3rd party integration port documentation as well